### PR TITLE
ENG-38 Implements Taurus Engine tool taurus-check-model-consistency

### PR DESCRIPTION
**Pull request by vitaly-krugl**
CC @scottpurdy, @oxtopus 

This tool validates parity between models (and their attributes) in Taurus Engine repository and DynamodDB.

ENG-38 Implemented tool check_model_consistency.py for checking consistency between models in Taurus Engine sql database and DynamodDB

ENG-38 Implemented unit tests for check_model_consistency.py

### CODE REVIEW

File: taurus/taurus/engine/check_model_consistency.py [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+        result code for warnings just like for errors.
+  """
+  parser = argparse.ArgumentParser(description=__doc__)
+
+  parser.add_argument(
+    "-v",
+    "--verbose",
+    action="store_true",
+    dest="verbose",
+    help=("Turn on verbose mode."))
```
Comment (oxtopus) [inline taurus/taurus/engine/check_model_consistency.py]: Parens here and below are unnecessary.

File: taurus/taurus/engine/check_model_consistency.py [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+    table_name=MetricDynamoDBDefinition().tableName,
+    connection = dynamodb_service.DynamoDBService.connectDynamoDB()
+  )
+
+  if verbose:
+    g_log.info("Accessing metrics in DynamodDB from %s via %r",
+               metricTable.table_name, metricTable.connection)
+
+  resultSet = retryOnTransientDynamoDBError(g_log)(metricTable.scan)()
+  return tuple(row for row in resultSet)
```
Comment (oxtopus) [inline taurus/taurus/engine/check_model_consistency.py]: Is the comprehension necessary?  I think you can accomplish the same with `return tuple(resultSet)`.

**Comment by oxtopus** [comment]
:+1:

**Comment by vitaly-krugl** [comment]
Thanks @oxtopus, addressed via #548

**Pull request merged**